tests/OSPSuite.HelpersForTests/ModuleHelperForSpecs.cs

```diff
@@ -158,10 +160,82 @@ private IParameter newConstantParameter(string name, double value, ParameterBuil
             .WithMode(parameterBuildMode);
       }
 
+      //Module 2 will introduce
```

Comment:
@Yuri05 @rwmcintosh @PavelBal  Interested in added simple use cases here
Can we validate that this is the expected behavior?